lib/ui/page/home/page/chat/gallery/view.dart

```diff
@@ -0,0 +1,363 @@
+            ),
+          );
+        });
+      },
+    );
+
+    if (result == true) {
+      await c.reportChat();
+    }
+  }
+
+  /// Builds [body] for this page.
+  Widget _body(BuildContext context, GalleryController c) {
+    return switch (c.viewMode) {
+      GalleryViewMode.files => _filesBody(context, c),
+      GalleryViewMode.media => _mediaBody(context, c),
+    };
+  }
+
+  /// Builds empty [body] for this page.
+  Widget _emptyBody(BuildContext context, GalleryController c) {
+    final Style style = Theme.of(context).style;
+    return Column(
+      mainAxisAlignment: PlatformUtils.isMobile
+          ? MainAxisAlignment.start
+          : MainAxisAlignment.center,
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        if (PlatformUtils.isMobile) SizedBox(height: 30),
+        SvgIcon(switch (c.viewMode) {
+          GalleryViewMode.files => SvgIcons.galleryNoFiles,
+          GalleryViewMode.media => SvgIcons.galleryNoMedia,
+        }),
+        SizedBox(height: 20),
+        Text(
+          switch (c.viewMode) {
+            GalleryViewMode.files => 'label_no_files'.l10n,
+            GalleryViewMode.media => 'label_no_media'.l10n,
+          },
+          textAlign: TextAlign.center,
+          style: style.fonts.medium.regular.secondary,
+        ),
+      ],
+    );
+  }
+
+  /// Returns avaliable [actions] for [menu].
+  List<Widget> _actions(BuildContext context, GalleryController c) => [
+    ContextMenuRegion(
+      actions: [
+        ContextMenuButton(
+          label: 'btn_go_to_chat'.l10n,
+          trailing: const SvgIcon(SvgIcons.chat19),
+          inverted: const SvgIcon(SvgIcons.chat19White),
+          onPressed: c.goToChat,
+        ),
+        ContextMenuButton(
+          label: 'btn_report'.l10n,
+          trailing: const SvgIcon(SvgIcons.report),
+          inverted: const SvgIcon(SvgIcons.reportWhite),
+          onPressed: () => _reportChat(context, c),
+        ),
+      ],
+      enablePrimaryTap: true,
+      child: Padding(
+        padding: const EdgeInsets.fromLTRB(18, 20, 18, 20),
+        child: SvgIcon(SvgIcons.more),
+      ),
+    ),
+  ];
+
+  /// Returns [body] for file [attachment]s view.
+  Widget _filesBody(BuildContext context, GalleryController c) {
+    final Style style = Theme.of(context).style;
+
+    final Widget itemsList = Obx(
+      () => ListView.separated(
+        controller: c.scrollController,
+        padding: const EdgeInsets.all(10),
+        itemBuilder: (BuildContext context, int index) => DataAttachment(
+          c.items[index],
+          onPressed: () => c.downloadFile(c.items[index] as FileAttachment),
+        ),
+        separatorBuilder: (BuildContext context, int index) => Divider(),
+        itemCount: c.items.length,
+      ),
+    );
+
+    return LayoutBuilder(
+      builder: (context, constraints) => !PlatformUtils.isMobile
+          ? Block(
+              expanded: false,
```

Comment:
Looks unreadable due to the large `=>` + ternary operator. Can't we refactor that into a little bit different structure, please?

Something like that:

```dart
return LayoutBuilder(
  builder: (context, constraints) {
    if (PlatformUtils.isMobile) {
       return ...;
    }

    return ...;
  }
);
```

Wouldn't that make a more readable code? I suggest to follow that approach elsewhere as well, please. Instead of doing large `=>` let's do `{ ... }`. Formatter should do a better job at handling the code blocks better as well.